src/ui/public/styles/base.less

```diff
@@ -1078,4 +1078,225 @@ fieldset {
   }
 }
 
+
+// TODO: Extract these styles into the UI Framework.
+
```

Comment:
yes, this is what i am referring to. CJ wrote this styles. how they became part of this PR ? they are not in master or in UI framework yet, and i need them here so i added them as first commit. maybe i should move them to .less file under kbn_vislib_vis_types ?